Sources/Clappr/Classes/Base/Loader.swift

```diff
@@ -1,63 +1,46 @@
 open class Loader {
-    internal(set) open var playbackPlugins: [Plugin.Type] = [AVFoundationPlayback.self, NoOpPlayback.self]
+    
+    static let shared = Loader()
+    
+    internal(set) open var playbacks: [Plugin.Type] = [AVFoundationPlayback.self, NoOpPlayback.self]
     #if os (iOS)
     internal(set) open var containerPlugins: [Plugin.Type] = [PosterPlugin.self, SpinnerPlugin.self]
     #else
     internal(set) open var containerPlugins: [Plugin.Type] = []
     #endif
     internal(set) open var corePlugins = [Plugin.Type]()
-
-    internal(set) open var mediaControl: MediaControl.Type = MediaControl.self
-
-    fileprivate var externalPlugins = [Plugin.Type]()
-
-    public convenience init() {
-        self.init(externalPlugins: [])
-    }
-
-    public init(externalPlugins: [Plugin.Type], options: Options = [:]) {
-        self.externalPlugins = externalPlugins
-
-        loadExternalMediaControl(options)
-
-        if !externalPlugins.isEmpty {
-            addExternalPlugins(externalPlugins)
-        }
-
+    
+    var externalPlugins: [Plugin.Type] = []
+    
+    private init() {
         Logger.logInfo("plugins:" +
-            "\n - playback: \(playbackPlugins.map({ $0.name }))" +
+            "\n - playback: \(playbacks.map({ $0.name }))" +
             "\n - container: \(containerPlugins.map({ $0.name }))" +
-            "\n - core: \(corePlugins.map({ $0.name }))" +
-            "\n - mediaControl: \(mediaControl)", scope: "\(type(of: self))")
-    }
-
-    fileprivate func loadExternalMediaControl(_ options: Options) {
-        if let externalMediaControl = options[kMediaControl] as? MediaControl.Type {
-            mediaControl = externalMediaControl
-        }
+            "\n - core: \(corePlugins.map({ $0.name }))")
     }
-
+    
     open func addExternalPlugins(_ externalPlugins: [Plugin.Type]) {
-        self.externalPlugins = externalPlugins
```

Comment:
Let's change this to `registerPlugin`, as we documented in the plugin loading diagram. This shouldn't apply only to external plugins, but instead all plugins. Built-in plugins should use the same process as externals.

Reply:
Done!